docs/src/operations/configuration.md

```diff
@@ -114,3 +114,18 @@ Action:
 Update your application's configuration
 ```
 
+# Logging
```

Comment:
Optional comments:
- not sure about "Logging" as top level header, I would probably move it one level down
- The last section on Google Stackdriver gets lost; would move it above 'Default Logging Configuration'